app/views/fsss_view.py

```diff
@@ -24,8 +24,7 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     line_manager = format_line_manager(current_job_role)
 
-    
-    employee_name = map_employee_name(employee_info,calledFrom="FSS - employeename")
+    employee_name = map_employee_name(employee_info)
```

Comment:
You seem to have removed the parameter here, but not from the map_employee_name function's arguments.

Reply:
*oops*
Have corrected this now